src/utilities/theme/utils.ts

```diff
@@ -22,15 +22,19 @@ export function buildCustomProperties(
   newDesignLanguage: boolean,
   tokens?: Record<string, string>,
 ): CustomPropertiesLike {
-  const {colors = {}, colorScheme, config} = themeConfig;
+  const {colors = {}, colorScheme, config, frameOffset = '0px'} = themeConfig;
```

Comment:
Providing this default ensures that `calc` won't break when nothing is passed as an offset. Either this or we fall back to 0px in every calc in Frame.scss i.e. `var(--p-frame-offset, 0px)`